rtdata/languages/default

```diff
@@ -1943,6 +1943,16 @@ PREFERENCES_LENSFUNDBDIR;Lensfun database directory
 PREFERENCES_LENSFUNDBDIR_TOOLTIP;Directory containing the Lensfun database.  Leave empty to use the default directories.
 PREFERENCES_LENSPROFILESDIR;Lens profiles directory
 PREFERENCES_LENSPROFILESDIR_TOOLTIP;Directory containing Adobe Lens Correction Profiles (LCPs)
+PREFERENCES_MAX_ZOOM_TITLE;Maximal zoom limit
+PREFERENCES_MAX_ZOOM_VARIANT_PERCENT_100;100%
+PREFERENCES_MAX_ZOOM_VARIANT_PERCENT_200;200%
+PREFERENCES_MAX_ZOOM_VARIANT_PERCENT_300;300%
+PREFERENCES_MAX_ZOOM_VARIANT_PERCENT_400;400%
+PREFERENCES_MAX_ZOOM_VARIANT_PERCENT_500;500%
+PREFERENCES_MAX_ZOOM_VARIANT_PERCENT_600;600%
+PREFERENCES_MAX_ZOOM_VARIANT_PERCENT_700;700%
+PREFERENCES_MAX_ZOOM_VARIANT_PERCENT_800;800%
+PREFERENCES_MAX_ZOOM_VARIANT_PERCENT_1600;1600%
```

Comment:
These aren't necessary, or maybe just one for the percent symbol if it's language dependent. For reference, this is the code for displaying the current zoom:
https://github.com/Beep6581/RawTherapee/blob/72bf271214cdfb2648ef3eeb8852d8c861a625d8/rtgui/zoompanel.cc#L141-L147